config/config.aws.reference.hocon

```diff
@@ -189,6 +189,10 @@
     "iglu:com.acme/skipped4/jsonschema/*-*-*"
   ]
 
+  # -- Whether the output parquet files should declare nested fields as non-nullable according to the Iglu schema.
+  # -- Set this to false if you use a query engine that dislikes non-nullable nested fields of a nullable struct.
+  "respectNullability": true
```

Comment:
Whoever reviews this, please comment on the name of this new config option.  And whether the description of this option is helpful.

The new config option will be documented on the snowplow docs site, but the description will probably just be copied over from this example config file.

Reply:
Do I understand it correctly:

* when `true` - use nullability definition from Iglu Schema
* when `false` - always makes nested fields nullable
?

Maybe we could include such `when true/false, then....` in description? Also about this fragment:

`Whether the output parquet files should declare nested fields as non-nullable according to the Iglu schema...`

it suggests that for `true` nested fields are always non-nullable, but they still can be nullable if this is what we have in Iglu schema...?

Another ideas for variants of the setting's name:

* `forceNullability`
* `alwaysNullable`
* `respectSchemaNullability`
*  `respectIgluNullability`